tools/perf/ResultsComparer/src/ResultsComparer/CommandLineOptions.cs

```diff
@@ -46,13 +52,13 @@ public static IEnumerable<Example> Examples
             get
             {
                 yield return new Example(@"Compare the results stored in 'C:\results\win' (base) vs 'C:\results\unix' (diff) using 5% threshold.",
-                    new CommandLineOptions { BasePath = @"C:\results\win", DiffPath = @"C:\results\unix", StatisticalTestThreshold = "5%" });
+                    new CommandLineOptions { Comparer = "bdn", BasePath = @"C:\results\win", DiffPath = @"C:\results\unix", StatisticalTestThreshold = "5%" });

```

Comment:
Maybe we could change these examples that have paths for win for Windows and unix for Unix to paths more relevant to OData projects